src/pages/guides/Webhooks/index.md

```diff
@@ -0,0 +1,195 @@
+
+**Example payload**
+
+```{
+  "account": {
+    "id": "6f70f1bd-7e89-4a7e-b4d3-7e576585a181"
+  },
+  "project": {
+    "id": "7e46e495-4444-4555-8649-bee4d391a997"
+  },
+  "resource": {
+    "id": "d3075547-4e64-45f0-ad12-d075660eddd2",
+    "type": "asset"
+  },
+  "type": "file.ready",
+  "user": {
+    "adobe_auth_id": "4B631AB66789A9A20A494025@c62f24cc5b5b7e0e0a494004",
+    "id": "56556a3f-859f-4b38-b6c6-e8625b5da8a5"
+  },
+  "workspace": {
+    "id": "378fcbf7-6f88-4224-8139-6a743ed940b2"
+  }
+}
+```
+
+> **Note**: The `adobe_auth_id` will not always be present, depending on whether the user performing the action is signed in via Adobe sign-in or not.
+
+In the above example of a `file.created` event, the `resource.id` indicates the `ID` of the newly created Asset. Additionally, `team`, `project`, and `user` objects are included.
+
+These resource identifiers indicate the `team.id`, `project.id` and `user.id` of the resource that the webhook relates to, and can be used to filter events on the receiving end of the incoming webhook without having to make an API call to look up the resource. If you've implemented any sort of caching of those resources, you can also perform a local look up against your cache without resorting an additional API call.
+
+> **We do not include any additional information beyond the resource id about the subscribed resource**. If your application requires additional information or context, we recommend making an API call to look-up more information about the resources being referenced.
+
+## Webhook Event Subscriptions
+
+When creating and updating webhooks, identify which events you’re interested in. Choose as few or as many as you’d like, note that the experience is better if you subscribe to fewer events, splitting up your webhooks logically with different naming schemes and different endpoints so that you can model your business logic on the receiving end to do less filtering and routing in shared functions.
+
+### Projects
+
+|Event	|Description	|
+|---	|---	|
+|`project.created`	|A new Project has been **created**	|
+|`project.updated`	|A Project's settings are **updated**	|
+|`project.deleted`	|A Project has been **deleted**	|
+
+### Files
+
+|Event	|Description	|
+|---	|---	|
+|`file.created`	|An File has been **created** in Frame.io. This does trigger before the file has actually been uploaded though so it will likely trigger before the file is fully uploaded - so keep that in mind dependeing on what you intend to do once you receive this event	|
+|`file.ready`	|All transcodes have **completed**, after an file has been uploaded and processed	|
+|`file.updated`	|A Files’ description, name, or other file information is changed	|
+|`file.deleted`	|A File is **deleted** (manually or otherwise)	|
```

Comment:
Is the "otherwise" comment intended to cover lifecycle-based deletion? If so, that's not yet supported in v4, and I think we should consider adding a couple of asset lifecycle specific webhooks so that you could get a warning as well via webhook, and then choose whether to act on it or not.